### ISSUE TITLE
Single-line CodeMirror Autocomplete shortcuts doesn't work on macOS

### ISSUE THREAD

**Issue by lukashornych** [lukashornych/evitalab#133]
CodeMirror autocomplete mouse and keyboard navigation doesn't work on macOS properly. It prevents users from using the autocompletion fully. Probably bug in the autocomplete plugin itself.
